core/capabilities/ccip/ccip_integration_tests/rmn/rmn_home_test.go

```diff
@@ -0,0 +1,172 @@
+		minObservers2   = 0
+		observerBitmap2 = 1
+	)
+
+	//================================Deploy and configure RMNHome===============================
+	rmnHomeAddress, _, rmnHome, err := rmn_home.DeployRMNHome(uni.Transactor, uni.Backend)
+	require.NoError(t, err)
+	uni.Backend.Commit()
+
+	staticConfig, dynamicConfig := integrationhelpers.GenerateRMNHomeConfigs(
+		"PeerID1",
+		"DummyPublicKey1",
+		"This is a sample offchain configuration in the static config",
+		chainID1,
+		minObservers1,
+		big.NewInt(observerBitmap1),
+	)
+
+	_, err = rmnHome.SetCandidate(uni.Transactor, staticConfig, dynamicConfig, zeroBytes)
+	require.NoError(t, err)
+	uni.Backend.Commit()
+
+	configDigest, err := rmnHome.GetCandidateDigest(&bind.CallOpts{})
+	require.NoError(t, err)
+
+	_, err = rmnHome.PromoteCandidateAndRevokeActive(uni.Transactor, configDigest, zeroBytes)
+	require.NoError(t, err)
+	uni.Backend.Commit()
+
+	rmnHomeBoundContract := types.BoundContract{
+		Address: rmnHomeAddress.String(),
+		Name:    consts.ContractNameRMNHome,
+	}
+
+	err = uni.HomeContractReader.Bind(testutils.Context(t), []types.BoundContract{rmnHomeBoundContract})
+	require.NoError(t, err)
+
+	rmnHomeReader := readerpkg.NewRMNHomePoller(uni.HomeContractReader, rmnHomeBoundContract, lggr, 1*time.Millisecond)
+
+	err = rmnHomeReader.Start(testutils.Context(t))
+	require.NoError(t, err)
+
+	t.Cleanup(func() {
+		err1 := rmnHomeReader.Close()
+		require.NoError(t, err1)
+	})
+
+	//================================Test RMNHome Reader===============================
+	expectedNodesInfo := integrationhelpers.GenerateExpectedRMNHomeNodesInfo(staticConfig, chainID1)
+
+	require.Eventually(
+		t,
+		assertRMNHomeNodesInfo(t, rmnHomeReader, configDigest, expectedNodesInfo, nil),
+		5*time.Second,
+		1*time.Millisecond,
+	)
+
+	// Add a new candidate config
+	staticConfig2, dynamicConfig2 := integrationhelpers.GenerateRMNHomeConfigs(
+		"PeerID2",
+		"DummyPublicKey2",
+		"This is a sample offchain configuration in the static config 2",
+		chainID2,
+		minObservers2,
+		big.NewInt(observerBitmap2),
+	)
+
+	_, err = rmnHome.SetCandidate(uni.Transactor, staticConfig2, dynamicConfig2, zeroBytes)
+	require.NoError(t, err)
+	uni.Backend.Commit()
+
+	candidateConfigDigest, err := rmnHome.GetCandidateDigest(&bind.CallOpts{})
+	require.NoError(t, err)
+
+	expectedCandidateNodesInfo := integrationhelpers.GenerateExpectedRMNHomeNodesInfo(staticConfig2, chainID2)
+
+	require.Eventually(
+		t,
+		assertRMNHomeNodesInfo(t, rmnHomeReader, candidateConfigDigest, expectedCandidateNodesInfo, nil),
+		5*time.Second,
+		1*time.Millisecond,
+	)
+
+	// Promote the candidate config
+	_, err = rmnHome.PromoteCandidateAndRevokeActive(uni.Transactor, candidateConfigDigest, configDigest)
+	require.NoError(t, err)
+	uni.Backend.Commit()
+
+	require.Eventually(
+		t,
+		assertRMNHomeNodesInfo(t, rmnHomeReader, candidateConfigDigest, expectedCandidateNodesInfo, &configDigest),
+		5*time.Second,
+		1*time.Millisecond,
```

Comment:
When these are too fast, they can starve the CPU and prevent it from making progress on the operation (particularly if accessing a lock). Fortunately, the tradeoff for using a longer duration is at most the latency from a single iteration, so the effect is minimal:
```suggestion
		100*time.Millisecond,
```

Reply:
Good remark, thanks.

Reply:
Could you bump the value on the other `Eventually` calls too?